includes/object-cache.php

```diff
@@ -1299,7 +1299,8 @@ protected function add_multiple_at_once( array $data, $group = 'default', $expir
             }, $tx->{$method}() ?: [] );
 
             if ( count( $results ) !== count( $keys ) ) {
-                throw new Exception( 'Redis pipeline returned unexpected result' );
+                $tx->discard();
+                return [];
```

Comment:
WP's API is a bit stupid, but I think these should stick with the expected return value:

```php
array_fill_keys($keys, false);
```

Reply:
Of course. Updated :)